### PR TITLE
docs: Main title and small general improvements

### PR DESCRIPTION
**Pull request by teknotica**
Hello!

Love the initiative on this project!

I'm a Spanish native speaker and noticed that the main title didn't make much sense (comparing with the English one). 

I also took the time to read and modified some other words.

Thanks!

### CODE REVIEW

**Comment by damianopetrungaro** [comment]
Hey, @teknotica ty for the effort!
The content looks good to me, but I'd ask you to change come CSS too, in order to fix this:

screen shot 2018-10-18 at 11 12 37 pm

Those 3 rules with a `width:80%` should be fine.
```
welcome__title
welcome__description
welcome__actions
```

Are you able to do it?

File: themes/conventional-commits/static/css/scss/layout/_welcome.scss [hunk 1]
```diff
@@ -41,13 +45,13 @@
...
     margin-right: $gap-md;
-    transition: background-color .3s ease, color .3s ease;
+    transition: background-color 0.3s ease, color 0.3s ease;
     &:last-child {
       margin-right: 0;
     }
     &:hover {
       background-color: $welcome-action-color;
-      color: get-color-accessible-for-background($welcome-action-color)
+      color: get-color-accessible-for-background($welcome-action-color);
```
Comment (teknotica) [inline themes/conventional-commits/static/css/scss/layout/_welcome.scss]: This was my `Prettier` - I can roll back if requested.

Comment (damianopetrungaro) [reply themes/conventional-commits/static/css/scss/layout/_welcome.scss]: Yes please, let the style as it is, edit just what needed 😄

**Comment by damianopetrungaro** [comment]
Hey @teknotica  sorry for the delay.
I see you changed the compiled css and js, that is done by the Makefile.
Could you please just revert those 2  compiled files? I'll prepare a PR to un-track them tomorrow
ty for the patience

**Comment by damianopetrungaro** [comment]
@teknotica I just removed them, rebase the master and push here.
I'll +1 it on the fly 😄

**Comment by teknotica** [comment]
Thanks @bcoe & @damianopetrungaro - I have synched my branch with your latest changes. Hope it works 😄

**Comment by damianopetrungaro** [comment]
Looks like you rebased in a strange way hmm

screen shot 2018-10-25 at 3 40 53 pm + 4 commits in total

**Comment by teknotica** [comment]
@damianopetrungaro I synched my `local master` with your `master` and merged it in (not rebased) into my branch.

**Comment by damianopetrungaro** [comment]
As you can see
```
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.
```
So you need to rebase properly I guess  :D 

If you need help doing it ping me ;)

**Comment by teknotica** [comment]
Hey @damianopetrungaro - right, I have rebased my changes. _Hopefully_ this time will work? 🙈  let me know.

**Comment by damianopetrungaro** [comment]
Looks amazing!
Ty for the effort! Really appreciated!

**Comment by teknotica** [comment]
Thanks @damianopetrungaro! I appreciate it 🤓

**Pull request merged**